Source/RomanNumeral.h

```diff
@@ -12,6 +12,29 @@
  * @param input The number to convert.
  * @param output String buffer in which to store the converted Roman numeral.
  */
+
+typedef struct {
+    int num;
+    const char * romanNumeral;
+} TableEntry_t;
+
+const TableEntry_t romanNumeralTable[] = {
+    {1000, "M"},
+    {900, "CM"},
+    {500, "D"},
+    {400, "CD"},
+    {100, "C"},
+    {90, "XC"},
+    {50, "L"},
+    {40, "XL"},
+    {10, "X"},
+    {9, "IX"},
+    {5, "V"},
+    {4, "IV"},
+    {1, "I"}
+};
+
+void Append(char *output, const char *toAppend);
```

Comment:
This too should only be in the C file (and declared `static`). This ensures that it's private to the module. This is important so that 1) we don't pollute the global namespace with functions that have a generic name and 2) so that it's more clear to clients of this library what the public API is.